package.json

```
@@ -4,6 +4,7 @@
   "description": "A Javascript function for reading a variety of coordinate formats and converting to decimal numbers. Builds on other efforts by returning the verbatim coordinates and the decimal coordinates all in one object.",
   "type": "module",
   "main": "merge.js",
+  "types": "merge.d.ts",
```

```suggestion
  "main": "./dist/cjs/merge.js",
  "types": "./dist/cjs/merge.d.ts",
```

Neither `main` nor the proposed `types` point to existing files. In the spirit of this PR adding maximal backward compatibility, this fixes things both for old TS versions and old Node.js / bundler versions.